installing/installing_bare_metal_ipi/ipi-install-postinstallation-configuration.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="ipi-install-post-installation-configuration"]
+[id="ipi-install-postinstallation-configuration"]
 = Installer-provisioned postinstallation configuration
 include::_attributes/common-attributes.adoc[]
-:context: ipi-install-post-installation-configuration
+:context: ipi-install-postinstallation-configuration
```

Comment:
This change should not be done because it will cause xrefs to break. I recommend that you revert it.

Reply:
Also revert the file name change too!